Tests/AlicerceTests/Utils/GeneratableObjectCacheTestCase.swift

```diff
@@ -0,0 +1,149 @@
+//
+//  GeneratableObjectCacheTestCase.swift
+//  Alicerce
+//
+//  Created by Filipe Lemos on 02/05/2018.
+//  Copyright © 2018 Mindera. All rights reserved.
+//
+
+import XCTest
+@testable import Alicerce
+
+extension String: GeneratableObjectKey {
```

Comment:
Although this is a unit test, I'm still not sure about having a `generate` method in the `String` namespace that generates a `DateFormatter` using as format the string contents. Since I don't believe such practice should be done in production code, having has an example might give the wrong impression as well.

Reply:
I think you're right. The original method was `dateFormatter` and it made sense to have it as an extension of `String`, but not as much now.

I share the opinion we should remove that test as it doesn't bring much value as it is and might be interpreted as a bad example.